main.go

```diff
@@ -10,28 +10,35 @@ import (
 	"strings"
 	"text/template"
 
-	"github.com/drone/drone-plugin-go/plugin"
+	"github.com/urfave/cli"
```

Comment:
Regarding the previously mentioned preference to `spf13/cobra`, many existing drone v0.5+ plugins use a certain `github.com/urfave/cli`, so it was very convenient for me to model this PR on those plugins and also use `urfave`.